code/game/machinery/computer/launchpad_control.dm

```diff
@@ -256,8 +167,110 @@ DEFINE_BUFFER_HANDLER(/obj/machinery/computer/launchpad)
 			. = TRUE
 		if("launch")
 			teleport(usr, current_pad, TRUE)
+			if (recall_time)
+				addtimer(CALLBACK(src, PROC_REF(teleport), usr, current_pad, FALSE), recall_time * 10)
 			. = TRUE
 
 		if("pull")
 			teleport(usr, current_pad, FALSE)
+			if (recall_time)
+				addtimer(CALLBACK(src, PROC_REF(teleport), usr, current_pad, TRUE), recall_time * 10)
```

Comment:
Should pull really send the object back after the timer expires?

Reply:
I can't think of a reason why that would be a wanted or expected effect.